test/WebSites/TagHelpersWebSite/TagHelpers/PrettyTagHelper.cs

```diff
@@ -33,7 +33,8 @@ public class PrettyTagHelper : TagHelper
 
         public override void Process(TagHelperContext context, TagHelperOutput output)
         {
-            if (MakePretty.HasValue && !MakePretty.Value)
+            if (MakePretty.HasValue && !MakePretty.Value || 
+                output.TagName == null)
```

Comment:
deserves a code comment mentioning case that `PrettyTagHelper` and `ConditionTagHelper` target the same element.  can see the reasoning as both change in this review but that'll be lost once the changes are in.